internal/mq/mqimpl/rocksmq/server/rocksmq_impl.go

```diff
@@ -594,8 +594,22 @@ func (rmq *rocksmq) Produce(topicName string, messages []ProducerMessage) ([]Uni
 		msgID := idStart + UniqueID(i)
 		key := path.Join(topicName, strconv.FormatInt(msgID, 10))
 		batch.Put([]byte(key), messages[i].Payload)
+		var pLen int
+		if len(messages[i].Properties) != propertiesLength {
+			log.Warn("length of properties is not 1",
+				zap.Any("properties", messages[i].Properties))
+		}
+		for k, v := range messages[i].Properties {
```

Comment:
All properties are serialized into a value, and the key is used as `/topicname/${msgID}/properties` seems better

Reply:
I thought about this method, but serialization may take more time？